test/bandit.spawn.test.js

```diff
@@ -33,6 +33,17 @@ describe('Bandit runner', () => {
     expect(report.annotations.length).toEqual(0)
   })
 
+  test('Handles when there are invalid and valid python files', async () => {
+    let mixedReport = await bandit('test/fixtures/python', ['sum.invalid.py', 'https.py'])
+
+    expect(mixedReport.annotations.length).toBe(4)
+    expect(mixedReport.annotations[0].title).toBe('B309:blacklist')
+    expect(mixedReport.annotations[3].path).toEqual('sum.invalid.py')
+    expect(mixedReport.annotations[3].start_line).toBe(1)
+    expect(mixedReport.annotations[3].annotation_level).toBe('failure')
+    expect(mixedReport.annotations[3].title).toBe('ERROR:Syntax error')
```

Comment:
How come this doesn't fail? I'd expect `Error: Syntax error` not `ERROR:Syntax error`

Reply:
Well, right now we do isssue_id:issue_name and there is no space. I sent a pull request where I add spaces.